packages/mosaic/tree/tree-selection.component.ts

```diff
@@ -174,12 +169,19 @@ export class McTreeSelection extends CdkTree<McTreeOption>
         differs: IterableDiffers,
         changeDetectorRef: ChangeDetectorRef,
         @Attribute('tabindex') tabIndex: string,
-        @Attribute('multiple') multiple: string
+        @Attribute('multiple') private multipleAttr: string
     ) {
         super(differs, changeDetectorRef);
 
         this.tabIndex = parseInt(tabIndex) || 0;
-        this.multiple = multiple === null ? false : coerceBooleanProperty(multiple);
+
+        if (this.multipleAttr !== null) {
```

Comment:
You set `multipleMode` here and also you do it in `set multiple(value: boolean)`

Why do you set `multipleMode` in two places ?

Reply:
the number of components use `set multiple(value: boolean)` and they should do not know anything about modes and etc they only want to use it in multiple mode with default behavior.
By our discussion we should have three states: multiple without any params (means default multiple mode) and two multiple modes when multiple is set as kyboard or checkbox

Reply:
ok, i will rewrite it.